AzTS Data Table/Readme.md

```diff
@@ -0,0 +1,273 @@
+# AzTS Data Table
+
+
+## **Overview**
+
+Azure Tenant Security Solution centrally scans subscriptions and get control scan result, processed RBAC data, resource inventory data, subscription meta data etc. and push all the data into a blob storage of central storage account.
+
+## **AzTS Data Table details**
+
+| Table Name | Description |
+|--|--|
+| [RBAC data](Readme.md#rbac-data)  | Role Based Access Control (RBAC) contains all PIM, permanent and classic role assignments details. |
+| [Baseline controls data](Readme.md#baseline-control-data) | Baseline Controls data  contains list of all controls (both baseline & non-baseline) supported by Azure Tenant Security (AzTS).   |
+| [Control result data](Readme.md#control-result-data)  | Control result data  contains evaluated control scan result with appropriate status reason. |
+| [Resource Inventory data](Readme.md#resource-inventory-data)|Resource inventory data  contains details of all resources present in subscription. It includes resource name, resource id, resource type, resource group name etc. |
+| [Subscription metadata](Readme.md#subscription-metadata)| Subscription metadata  contains each subscription details like subscription id, tenant id, state etc. |
+| [Assessment data](Readme.md#assessment-data)| Assessment Details  contains the evaluated assessment details with the appropriate status reason.|
+| [Policy State data](Readme.md#policy-state-data) | Policy State  contains the evaluated Policy State details with the assigned scope and it's state.|
+| [Role definition data](Readme.md#role-definition-data)| Role Definition Details  contains both custom and built in roles information along with assigned scope.|
+| [Secure Score Control data](Readme.md#secure-score-control-data) | Secure Score Control  contains the secure score evaluated control details with the current score and healthy and unhealty resource count.|
+| [Secure Score data](Readme.md#secure-score-data) | Secure score  contains the evaluated secure score details current and maximum score. |
+___
+
+### **RBAC data**
+
+|Field Name|Field Type|Description|Sample Value| 
+|--|--|--|--|
+|JobId|int|Job ID for scan (YYYYMMDD).|20230414|
+|RoleName|string|The role assignment name.|Reader|
+|RoleId|string|The role assignment ID.|xx0x0xx0-0xx0-0000-0x00-000x0x00000x|
+|AccountType|string|The role assignment account type.|ServicePrincipal|
```

Comment:
Do we want to list all possible values in case of data fields which are basically enums or have fixed possible values?

Reply:
Same comment applies for all such fields.

Reply:
I don't think that is required to fetch data from blob storage to any other data sources. And if we add in sample value column that will add confusion

Reply:
You might want to confirm this with the team. I feel like for enums people might want to know all possible values.

Reply:
done